### ISSUE TITLE
`import-x/export`: False-positive with type export

### ISSUE THREAD

**Issue by silverwind** [un-ts/eslint-plugin-import-x#414]
foo.ts
```ts
export const foo = 1;
```

bar.ts
```ts
export type * from "./foo.ts"; // Error: Multiple exports of name 'foo'
export {foo} from "./foo.ts"; // Error: Multiple exports of name 'foo'
```

It seems the rule is unaware of type exports being distinct from value exports. The `export type *` does not include `foo` which is not a type, so the export is not duplicate.

**Comment by dackmin** [comment]
Same here, had to disable the rule as this issue has never been fixed and I explicitly want to export `foo` (as a function) from `foo.ts` for execution, but export all types at the same time.
